tasks/link-checker.js

```diff
@@ -179,5 +267,8 @@ module.exports = {
   },
   checkDevLinks: async () => {
     return await linkChecker('http://localhost:3000');
+  },
+  checkSpecificLinks: async (localDomain, links) => {
+    return await linkChecker(localDomain, links);
```

Comment:
Added this to help test locally instead of having to run through the whole sitemap. You can import this function in another file and pass in `'http://localhost:3000'` as the localDomain and pass it an array of urls for the linkchecker to run through and check the links on those pages